source/darwin/secure_transport_tls_channel_handler.c

```diff
@@ -830,22 +830,28 @@ static struct aws_channel_handler *s_tls_handler_new(
         goto cleanup_st_handler;
     }
 
+    OSStatus setProtocolStatus = errSecSuccess;
     switch (secure_transport_ctx->minimum_version) {
         case AWS_IO_SSLv3:
-            SSLSetProtocolVersionMin(secure_transport_handler->ctx, kSSLProtocol3);
+            setProtocolStatus = SSLSetProtocolVersionMin(secure_transport_handler->ctx, kSSLProtocol3);
+            AWS_LOGF_TRACE(AWS_LS_IO_TLS, "Attempted to set minimum SSL to: 3");
             break;
         case AWS_IO_TLSv1:
-            SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol1);
+            setProtocolStatus = SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol1);
+            AWS_LOGF_TRACE(AWS_LS_IO_TLS, "Attempted to set minimum TLS to: 1");
             break;
         case AWS_IO_TLSv1_1:
-            SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol12);
+            setProtocolStatus = SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol12);
+            AWS_LOGF_TRACE(AWS_LS_IO_TLS, "Attempted to set minimum TLS to: 1.1");
             break;
         case AWS_IO_TLSv1_2:
-            SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol12);
+            setProtocolStatus = SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol12);
+            AWS_LOGF_TRACE(AWS_LS_IO_TLS, "Attempted to set minimum TLS to: 1.2");
             break;
         case AWS_IO_TLSv1_3:
 #if TLS13_AVAILABLE
-            SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol13);
+            AWS_LOGF_WARN(AWS_LS_IO_TLS, "TLS 1.3 currently not supported. Will use system default TLS instead");
+            setProtocolStatus = SSLSetProtocolVersionMin(secure_transport_handler->ctx, kTLSProtocol13);
 #else
```

Comment:
The log statement doesn’t seem accurate. Should we update the Tls13_Available flag for now to indicate that it is not supported?

Reply:
Good idea! That would be better than logging or throwing a fatal error, as then it wouldn't break anything and would be more easily visible to customers.